apm-sniffer/apm-sdk-plugin/spymemcached-2.x-plugin/src/main/java/org/skywalking/apm/plugin/spymemcached/v2/define/MemcachedInstrumentation.java

```diff
@@ -0,0 +1,92 @@
+package org.skywalking.apm.plugin.spymemcached.v2.define;
+
+import static org.skywalking.apm.agent.core.plugin.match.NameMatch.byName;
+
+import java.util.List;
+
+import org.skywalking.apm.agent.core.plugin.interceptor.ConstructorInterceptPoint;
+import org.skywalking.apm.agent.core.plugin.interceptor.InstanceMethodsInterceptPoint;
+import org.skywalking.apm.agent.core.plugin.interceptor.enhance.ClassInstanceMethodsEnhancePluginDefine;
+import org.skywalking.apm.agent.core.plugin.match.ClassMatch;
+import org.skywalking.apm.logging.ILog;
+import org.skywalking.apm.logging.LogManager;
+
+import net.bytebuddy.description.method.MethodDescription;
+import net.bytebuddy.matcher.ElementMatcher;
+
+import static net.bytebuddy.matcher.ElementMatchers.named;
+import static net.bytebuddy.matcher.ElementMatchers.takesArgument;
+
+/**
+ * 
+ * {@link MemcachedInstrumentation} presents that skywalking intercept all constructors and methods of 
+ * {@link net.spy.memcached.MemcachedClient}.
+ * {@link XMemcachedConstructorWithInetSocketAddressListArgInterceptor} intercepts the constructor with
+ * argument {@link java.net.InetSocketAddress}.
+ * 
+ *@author IluckySi
+ * 
+ */
+public class MemcachedInstrumentation extends ClassInstanceMethodsEnhancePluginDefine {
+    
+    private static final ILog logger = LogManager.getLogger(MemcachedInstrumentation.class);
+    
+    private static final String ENHANCE_CLASS = "net.spy.memcached.MemcachedClient";
+    
+    /**
+     *  public MemcachedClient(ConnectionFactory cf, List<InetSocketAddress> addrs)
```

Comment:
What does the comment stand for? This is only a method name, I suppose.

Reply:
The comment is the method's detailed descriptor!

Reply:
I need to delete the comment for the method?

Reply:
@wu-sheng  Need I to delete the comment for the method?

Reply:
You should make the comment more clear. Not just put the method signature here.